zuul.d/nodesets.yaml

```diff
@@ -29,6 +29,12 @@
       - name: fedora-34
         label: fedora-34-1vcpu
 
+- nodeset:
+    name: fedora-latest-4vcpu
+    nodes:
+      - name: fedora-34
+        label: fedora-34-4vcpu
+
```

Comment:
what was failing on centos-8? I am not sure we want to move all testing to fedora, given it has a support window of 12 months.

I added centos-8-stream images on the weekend, does that nodeset work for you?

Reply:
we are planning to use tox for the kubernetes.core collection, but the centos-7 version of molecule is not up to date, I think we can upgrade it but I don't know the impact on others jobs running with.

Reply:
centos-8 has only 1 vcpu molecule run will timeout

Reply:
can you try centos-8-stream-1vcpu

I'll create 2vcpu now, so we can also test it.  I really want to get off fedora for testing as it is too fast moving.